### ISSUE TITLE
GNOME now includes an official dark theme variant

### ISSUE THREAD

**Issue by jurf** [axxapy/Adwaita-dark-gtk2#5]
Since `gnome-themes-standard 3.22` GNOME ships with an official dark GTK+ 2 theme. It would  maybe be a good idea to add a notice or something. Sorry I let you know so late but I just kept forgetting.

**Comment by ghost** [comment]
@DoctorJellyface Thanks for this info. I see indeed the dark GTK2 theme files, but I don't know how to use it. My GTK2 apps (like Gimp) are still light. Do you have an idea?

**Comment by quequotion** [comment]
The dark gtk2 theme gnome is shipping seems to be broken. I have "Adwaita-dark" specified everywhere gtk2 could be looking for it but all I get are ordinary bright-themed windows with unreadable black buttons (only button _backgrounds_ seem to get "dark"). In fact, I've tried to make several modifications to it and  gtk2 applications just ignore any settings in its gtkrc and keep the unreadable black buttons and ordinary bright theme. Pretty frustrating.

**Comment by jurf** [comment]
I’m sorry I’m sorry it’s all fixed now! I’ve been putting the release (it was my completely first one) off for so long I completely forgot to do it, and I didn’t have time either way, but it’s fixed, 3.22.3 contains a fully working fixed and supported GTK+2 theme. Won’t happen again (I hope)! Sorry for all the trouble!

**Comment by quequotion** [comment]
No worries! I did find an interim workaround.

I'm not sure if this makes any sense, but I think the broken colors were caused by something trying to put the GTK3 Adwaita colors on GTK2 windows (ignoring ":dark" and partially overriding the gtk2 settings). I ended up symlinking the gtk-3.0 directory from Adwaita into Adwaita-dark's folder, specifying my theme as "Adwaita-dark" universally and setting the options for gtk3 themes to prefer dark colors. Both gtk3 and gtk2 windows are properly themed now.
